output/endpoint.go

```diff
@@ -0,0 +1,5 @@
+//go:build !prod
```

Comment:
How does this react if we simply run `go run main.go` ?

Reply:
This `go:build` directive tells the compiler to include the file only if the build tag `prod` is not set, as a consequence the behavior of `go run ./cmd/runner` does not change. To summarize:

```sh
> go run ./cmd/runner
# endpoint is http://localhost:9998/report

> go run -tags prod ./cmd:runner
# endpoint is https://api.benchttp.app/report
```